prac_05/wimbledon.py

```diff
@@ -0,0 +1,34 @@
+FILENAME = "wimbledon.csv"
+def main():
+    """Read file"""
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        data = in_file.readlines()
```

Comment:
You could import the CSV module as it does some of the things you are doing manually in a single function